src/dotnet/commands/dotnet-install/xlf/LocalizableStrings.cs.xlf

```diff
@@ -98,6 +98,16 @@ Arguments: {1}
 Output: {2}{3}</target>
         <note />
       </trans-unit>
+      <trans-unit id="GlobalOptionDescription">
+        <source>Install user wide.</source>
+        <target state="new">Install user wide.</target>
+        <note />
+      </trans-unit>
+      <trans-unit id="InstallToolCommandOnlySupportGlobal">
+        <source>Install tool only support install user wide (-g).</source>
+        <target state="new">Install tool only support install user wide (-g).</target>
```

Comment:
@KathleenDollard this need your review. This is the message when the consumer forget to add `-g` to `dotnet install tool my.tool`. Do you think it is clear enough to guide them adding `-g` ?

Reply:
We'll check this at standup. I anticipate this is temporary as I think we're still planning session tools after Preview 1. 

The --global switch (-g) is currently required because only user wide tools supported.

@peterhuene can you look at that as well

Reply:
@wli3's PR properly checks for the presence of the global flag and aborts if not present.

Agree with @KathleenDollard's suggestion of making it clear that this is "currently required" because only user-wide is supported.  As a user encountering a message like that, I would think "oh okay, perhaps one day the -g option won't be always required".

I'd just amend the suggestion to: `The --global switch (-g) is currently required because only user wide tools are supported.` (added `are`).

Reply:
Yes, agree with Peter

Reply:
Fixed, change to above text